tcf_website/templates/reviews/review.html

```diff
@@ -36,48 +40,76 @@
-                            <i class="fa fa-hourglass-half fa-fw"></i>
-                            {{ review.hours_per_week }}
+                <div class="review-content mb-2">
+                  <div class="review-text collapse" id="reviewText{{review.id}}"
+                     aria-expanded="false" style="white-space: pre-line;">
+                    <p class="review-text-all">{{ review.text }}</p>
+                  </div>
+                  <a role="button" class="review-collapse-link collapsed"
+                     data-toggle="collapse" href="#reviewText{{review.id}}"
+                     aria-expanded="false" aria-controls="reviewText{{review.id}}">
+                  </a>
+                </div>
+
+                <div class="review-footer row mt-3">
+                    <!-- Review Ratings -->
+                    <div class="review-ratings col-sm-8 d-flex align-items-end">
+                        <div class="d-flex text-center flex-wrap mb-2">
+                            <div id="review-average" class="font-weight-bold">
+                                <i class="fa fa-star fa-fw"></i>
+                                {{ review.average|floatformat:2 }}
+                            </div>
+                            <div class="ml-4 text-muted">
+                                <i class="fas fa-thumbs-up"></i>
+                                {{ review.recommendability }}
+                            </div>
+                            <div class="ml-4 text-muted">
+                                <i class="fa fa-user"></i>
+                                {{ review.instructor_rating }}
+                            </div>
+                            <div class="ml-4 text-muted">
+                                <i class="far fa-smile-beam"></i>
+                                {{ review.enjoyability }}
+                            </div>
+                            <div class="ml-4 text-muted">
+                                <i class="fa fa-dumbbell fa-fw"></i>
+                                {{ review.difficulty }}
+                            </div>
+                            <div class="ml-4 text-muted">
+                                <i class="fa fa-hourglass-half fa-fw"></i>
+                                {{ review.hours_per_week }}
+                            </div>
                         </div>
                     </div>
 
-                    <div class="text-right">
-                        <p>{{ review.semester }}</p>
-                        <p id="date">{{ review.created|date }}</p>
+                    <div class="review-date col-sm-4 pl-0 text-right text-tcf-orange">
+                        <h6 class="font-weight-bold">{{ review.semester }}</h6>
+                        <h6 id="date">Updated {{ review.created|date }}</h6>
                     </div>
+
                 </div>
             </div>
         </div>
     </div>
 </div>
+
+<script type="module">
```

Comment:
What's our policy on using ES6 modules? I'm assuming these are used elsewhere in the codebase.

I ask because, if using ES6 modules is permissible, I think we should refactor Chart.js to be included through modules. (The linter complains about Chart.js otherwise.)

Reply:
hmm, I'm honestly not too familiar with them. could discuss during the meeting. @Bartleby2718 any thoughts?

Reply:
I don't know much about frontend either. I just googled this and that to make it work, and this is what I ended up with. Please contribute if you have any insight into this!